contrib/storage-kudu/src/main/java/org/apache/drill/exec/store/kudu/KuduStoragePlugin.java

```diff
@@ -35,15 +35,20 @@
   private final KuduSchemaFactory schemaFactory;
 
   private final KuduClient client;
-
   public KuduStoragePlugin(KuduStoragePluginConfig configuration, DrillbitContext context, String name)
-      throws IOException {
+    throws IOException {
     super(context, name);
     this.schemaFactory = new KuduSchemaFactory(this, name);
     this.engineConfig = configuration;
     this.client = new KuduClient.KuduClientBuilder(configuration.getMasterAddresses()).build();
   }
 
+
+  @Override
+  public void start() throws IOException {
+
+  }
```

Comment:
Is this change intentional? Or an artifact of rebasing? I suspect I removed this method in some PR.

Reply:
Please remove as @paul-rogers suggested.

Reply:
Fixed.